apache-rat-core/src/main/java/org/apache/rat/Report.java

```diff
@@ -405,7 +422,6 @@ static ReportConfiguration createConfiguration(String baseDirectory, CommandLine
                 }
 
                 URL url = Report.class.getClassLoader().getResource(String.format("org/apache/rat/%s.xsl", style[0]));
-
```

Comment:
Should we add a tryCatch around this trying to load the xsl file in case user input is invalid/not found?

Reply:
ok